.eslintrc.json

```diff
@@ -4,6 +4,16 @@
         "node": true
     },
     "extends": "eslint:recommended",
+    "ecmaFeatures": {
+      "sourceType": "module"
+    },
+    "parserOptions": {
+      "sourceType": "module",
+      "ecmaFeatures": {
+        "jsx": true,
+        "experimentalObjectRestSpread": true
+      }
+    },
```

Comment:
This code says: "the entire project should use ecma modules, but we don't. We only use the module import syntax inside the `client` + `tests/client` directories. Therefore it should be able to get removed.